string action # "goto" , "open" , "close"
int8 desired_floor
---
bool success
string message